File: templates/web/backend/python/fastapi/sources/health.py
from datetime import datetime, timezone

from fastapi import APIRouter

router = APIRouter()


@router.get("/api/health")
async def health():
    return {
        "status": "ok",
        "service": "{{project_name}}",
        "timestamp": datetime.now(timezone.utc).isoformat(),
    }

File: templates/web/backend/rust/actix-web/sources/health.rs
use actix_web::{web, HttpResponse};
use chrono::Utc;
use serde_json::json;

use crate::config::Config;

pub fn configure(cfg: &mut web::ServiceConfig) {
    cfg.route("/api/health", web::get().to(health_handler));
}

async fn health_handler(cfg: web::Data<Config>) -> HttpResponse {
    HttpResponse::Ok().json(json!({
        "status": "ok",
        "service": cfg.name,
        "timestamp": Utc::now().to_rfc3339()
    }))
}
